app/client/components/payment/update/card/cardInputForm.tsx

```diff
@@ -64,7 +67,17 @@ export const CardInputForm = (props: CardInputFormProps) => {
     });
   };
 
+  const loadStripe = () => {
+    import("@stripe/stripe-js").then(StripeAsync => {
+      setStripePromise(StripeAsync.loadStripe(props.stripeApiKey));
+    });
+  };
+
   const loadSetupIntent = (recaptchaToken: string) => {
+    if (stripePromise === null) {
+      loadStripe();
+    }
+
```

Comment:
this is not synchronous? think you might have a race condition?

Reply:
> this is not synchronous? think you might have a race condition?

I've taken some screenshots in order to explain the approach here ...

The first image is the default route (account overview) on the existing master branch
![01_master](https://user-images.githubusercontent.com/2510683/91311704-4ae27380-e7ab-11ea-8693-c9692ec50f83.png)


The 2nd image is if we move the loadStripe call within the useEffect hook but keep the existing stripe imports (again on the default route - account overview):
![02_loadstripeInUseEffect](https://user-images.githubusercontent.com/2510683/91311826-6f3e5000-e7ab-11ea-8966-c91a040ec7a4.png)

The 3rd and 4th images are the approach taken in this pr (the 1st image being the default route and the 2nd being the update payment page after capthca is complete)
![03_asyncLoadStripeImport](https://user-images.githubusercontent.com/2510683/91312146-d2c87d80-e7ab-11ea-96a4-9cb20e375315.png)

update payment page:
![04_asyncLoadStripeImport](https://user-images.githubusercontent.com/2510683/91312180-dbb94f00-e7ab-11ea-9266-a0a41cfa3297.png)